Grasshopper.fs

```diff
@@ -55,6 +55,11 @@ module Types =
           Assign of lvalue : ShadowVar * prim : GrassPrim
         | /// <summary>A pure assumption.</summary>
           PureAssume of assumption : BoolExpr<Sym<GrassVar>>
+        | /// <summary>A branch.</summary>
+          Branch of
```

Comment:
Propagating the existence of atomic branches into GRASShopper.  These are just emitted as standard if-then-else, since at the GRASShopper level everything is sequential.